scripts/check_configs.ts

```diff
@@ -681,10 +681,22 @@ async function main() {
 
       if (bridgesMap && bridgesMap.ibc.length && bridgesMap.polynetwork.length) {
         const polynetworkBridges = bridgesMap.polynetwork.map((bridge) => bridge.bridgeAddresses).flat()
-        const ibcBridges = bridgesMap.ibc.map((bridge) => bridge.bridgeAddresses).flat()
+        const ibcBridges = bridgesMap.ibc.map((bridge) => bridge.channels.src_channel)
         bridgesArr = polynetworkBridges.concat(ibcBridges)
       }
 
+      const res = await sdk.query.bridge.ConnectionAll({
+        bridgeId: new Long(0),
+        pagination: PageRequest.fromPartial({
+          limit: new Long(100000),
+        }),
+      })
+      const axelarBridges: string[] = res.connections.map(bridge => bridge.connectionId)
+
+      if (axelarBridges.length > 0) {
+        bridgesArr = bridgesArr.concat(axelarBridges)
+      }
```

Comment:
Don't think there's a need to query axelar bridges again, just use `bridges.axelar`